src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMFeatureQC.h

```diff
@@ -86,17 +86,17 @@ namespace OpenMS
 
       // Feature members
       /// retention time lower bound
-      double retention_time_l;
+      double retention_time_l { 0.0 };
```

Comment:
The addition of default values is a good idea.  We just want to be careful that in "filter" mode these default values to not lead to the exclusion of peaks without the user being aware.